app/templates/base.html

```diff
@@ -124,8 +124,9 @@ <h1>Stay in touch</h1>
         </div>
     </footer>
 
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
```

Comment:
interesting, why did you add jQuery?

Reply:
There was an error in the console log that about missing jQuery for `ekko-lightbox.min.js` at base.html line 129. So I added jquery. If the `ekko-lightbox.min.js` is not necessary then we can remove both of them

Reply:
yes, lets remote the jquery here, I will remove `ekko-lightbox.min.js` as well, it is not been used anyway

Reply:
All right, I've fixed that